### ISSUE TITLE
Computation of accuracy over WikiTableQuestions dataset

### ISSUE THREAD

**Issue by Sohanpatnaik106** [microsoft/Table-Pretraining#30]
Hey, just wanted to clarify whether the model for the task of table question answering on the WikiTableQuestions dataset was trained in a generative way or was is span-based.

If it is trained in a generative manner, then how is the accuracy computed? 

Thanks.

**Comment by SivilTaram** [comment]
@Sohanpatnaik106 Hi, sorry for missing your message! The model is trained in a generative way, not span-based. We use the symbol `, ` as the separator for different answers if there are multiple answers. And then compare the set with the official ground-truth set. Hope it answers your question!
